### PR TITLE
standardize config and app setup; rename `cmd/pds` to `cmd/laputa`

### PR DESCRIPTION
**Pull request by bnewbold**
We had discussed these changes briefly. Probably worth a one-over.

I'm fairly opinionated about using the `DATABASE_URL` convention, don't care as much about the other variables.

### CODE REVIEW

File: cmd/beemo/main.go [hunk 1]
```diff
@@ -113,7 +118,7 @@ func pollNewReports(cctx *cli.Context) error {
 	xrpcc.Auth.Did = auth.Did
 	xrpcc.Auth.Handle = auth.Handle
 
-	adminToken := cctx.String("admin-token")
+	adminToken := cctx.String("admin-password")
```
Comment (whyrusleeping) [inline cmd/beemo/main.go:121]: is it a password?

Comment (bnewbold) [reply cmd/beemo/main.go:121]: Yeah, and it was confusing initially when I thought it was a token. The way it works is HTTP `Basic` auth (not `Bearer`) with username `admin` and this as the password.

We are planning to switch and use regular auth (the JWT tokens, as HTTP `Bearer` auth), plus an admin flag on accounts, but this is how it works today.

Some interfaces, like redsky, do say "token" though.

**Comment by bnewbold** [comment]
Updated the `DATABASE_URL` helper to accept the `sqlite=` and `postgres=` (DSN) syntax from before. I don't have test coverage for this, but did test all the variations locally. Will end up with test coverage when I add postgres tests in CI.

@whyrusleeping said in slack "other than that it seems good"

**Review by whyrusleeping — APPROVED** [review]

**Comment by bnewbold** [comment]
I merged and pushed this, the ordering was weird and github was confused and marked as closed.

**Pull request closed without merging**